lib/process-services-cloud/src/lib/form/components/widgets/attach-file/upload-cloud.widget.ts

```diff
@@ -160,4 +162,40 @@ export class UploadCloudWidgetComponent extends WidgetComponent implements OnIni
     fileClicked(file: ContentLinkModel): void {
         this.formService.formContentClicked.next(file);
     }
+
+    isAlfrescoAndLocal(): boolean {
+        return this.field?.params?.fileSource?.serviceId === FileSourceTypes.ALL_FILE_SOURCES_SERVICE_ID;
+    }
+
+    isPathVariableType(type: string): boolean {
+        return this.field?.params?.fileSource?.destinationFolderPath?.type === type;
+    }
+
+    getDestinationFolderPathFromMappingVariable() {
```

Comment:
get is a word that you expect it to return something, can we use a different name other than get?